### PR TITLE
add photos programming-r; update description

### PR DESCRIPTION
**Pull request by 3mmaRand**

### CODE REVIEW

File: workshops/programming_r.qmd [hunk 1]
```diff
@@ -1,35 +1,38 @@
...
-  1-sentence summary of workshop.
-categories: [add, comma, separated, categories]
+  Improve your R programming skills and reduce the amount of duplication in your code.
+categories: [R, tidyverse, purrr, functions, iteration, programming]
 ---
 
 # Description
 
-Full workshop description goes here. Multi-paragraph ok.
+This is a one-day, hands-on workshop intermediate R users who use the tidyverse and want to improve reduce the amount of duplication in their code. You will learn the two main ways to reduce duplication: creating functions and using iteration.
```
Comment (ijlyttle) [inline workshops/programming_r.qmd]: ... want to improve and reduce ...

File: workshops/programming_r.qmd [hunk 2]
```diff
@@ -1,35 +1,38 @@
...
+
+We will use a tidyverse approach to cover function design and iteration. When writing a function you will learn how to deal with "data masking", give default values to arguments, give unspecified arguments and what side effects are and how to manage them. In the afternoon, you will learn how to iterate across the columns of a dataframe using `across()`, what "anonymous" functions are and how to read and write multiple files using **`purrr`**.
```
Comment (ijlyttle) [inline workshops/programming_r.qmd]: ... give unspecified arguments, what side effects are and how to manage them.

**Review by ijlyttle — COMMENTED** [review]
Thanks Emma 🎉 just a couple wee suggestions

File: workshops/programming_r.qmd (hunk 2 above)
Comment (3mmaRand) [reply workshops/programming_r.qmd]: good spots, thanks

**Pull request merged**